### ISSUE TITLE
Pixel ratio of fisheye images

### ISSUE THREAD

**Issue by kunifujiwara** [koito19960406/ZenSVI#19]
In the segmentation result for the fisheye image, the total pixel ratio for all categories is 0.785323076923075. It should be 1.0.
Shouldn't it be calculated on all pixels in the image when it should be calculated on all pixels within the fisheye circle?

**Comment by koito19960406** [comment]
Yes, I'll fix this issue by excluding surrounding white space when calculating the pixel ratio

**Comment by kunifujiwara** [comment]
Thanks!
